modnet/models.py

```diff
@@ -379,7 +378,7 @@ def fit_preset(
         val_losses = 1e20 * np.ones((len(presets),))
 
         num_nested_folds = 5
-        if isinstance(nested, int) and nested != True:
+        if nested:
```

Comment:
Following our discussion...

```suggestion
        num_nested_folds = nested
        if num_nested_folds <= 1:
            num_nested_folds = 5
```